extensions/import/src/wizard/pages/prosePreviewPage.ts

```diff
@@ -32,7 +32,12 @@ export class ProsePreviewPage extends ImportPage {
 	}
 
 	async start(): Promise<boolean> {
-		this.table = this.view.modelBuilder.table().component();
+		this.table = this.view.modelBuilder.table().withProperties<azdata.TableComponentProperties>({
+			data: null,
```

Comment:
typically we're using `undefined` instead of `null`.  Unfortunately these are actually different values in TypeScript.

though if these are optional properties I'd recommend not including them at all.  If they are not optional you may consider making them optional.  i.e. it's better to only need to specify the values you actually want to set.

Reply:
..I see below the properties aren't optional, so probably just changing `null` to `undefined` is fine.